internal/dag/dag.go

```diff
@@ -861,6 +861,7 @@ type ExtensionCluster struct {
 	LoadBalancerPolicy string
 
 	// RouteTimeoutPolicy specifies how to handle timeouts to this extension.
+	// Defines default response timeout unless the
```

Comment:
oops 😄

Reply:
Uhh :) I've now removed that unfinished sentense.  

I was originally going to point out there in comment that the response timeout can come from that struct (originating from `ExtensionService.spec.timeoutPolicy.response`) or it may be overwritten by value from `HTTPProxy.spec.virtualhost.authorization.responseTimeout`. This can be seen in [code here](https://github.com/projectcontour/contour/blob/2b7e6ccdc9fdcce787bab0e0bf3e753fd62a41b9/internal/dag/httpproxy_processor.go#L302-L306) and it was existing functionality but it confused me a bit - and without this logic, which I think is not described in API docs, the `RouteTimeoutPolicy` could be even removed from `ExtensionCluster`.